### ISSUE TITLE
aria-controls="drop" is not a valid aria attribute

### ISSUE THREAD

**Issue by timcosta** [froala/wysiwyg-editor#1792]
##### Expected behavior.
`aria-controls` should have a valid value.

##### Actual behavior.
The dropdown buttons on https://www.froala.com/wysiwyg-editor/examples/tabs have `aria-controls="drop"` on them. Based on https://www.w3.org/TR/wai-aria/states_and_properties#aria-controls, and testing by an auditor, `drop` is not a valid value in this context. 

##### Steps to reproduce the problem.
(Describe the steps to reproduce the problem here)

##### OS.
masOS 10.12.3

##### Browser.
Chrome 56.0.2924.87 (64-bit)
